### DISCUSSION TITLE
How to reference properties with one line?

### DISCUSSION THREAD

**Discussion by WesHoward** [stdword/logseq13-full-house-plugin#33] [Q&A] [answered]
I am struggling to make sense of some of the template logic for prooperties. Taking a step back with this template for a support ticket. I want a one liner that I can enter to get the ticket number and client name into their properties. How do I reference the properties for those in the renderer?

`- Ticket documentation
template:: testticket
sampleticketnumber::
sampleclientname::
	- problem description
	- solution`

**Comment by stdword** [comment, marked as answer]
@WesHoward, hello!

Just created this showcase:
<img width="800px" src="https://github.com/stdword/logseq13-full-house-plugin/assets/1984175/d360c4a6-149b-449e-ae7f-2c68979ac074"/>
Details are [here](https://github.com/stdword/logseq13-full-house-plugin/discussions/34).

Is this what you need?
